Emby.Server.Implementations/Library/Resolvers/Movies/MovieResolver.cs

```diff
@@ -128,10 +128,9 @@ public override Video Resolve(ItemResolveArgs args)
                 return movie?.ExtraType == null ? movie : null;
             }
 
-            // Owned items will be caught by the video extra resolver
```

Comment:
I think there was a second instance of this comment,  might be worth restoring the base call there?

Reply:
That one already returned null before the extras refactor (with a commented out older line of code). When I add testing for better coverage I'll revise that comment.